addons/binding/org.openhab.binding.evohome/ESH-INF/thing/channels.xml

```diff
@@ -37,5 +37,11 @@
         <category>heating</category>
         <state min="4.5" max="30" step="0.1" pattern="%.1f °C" readOnly="false"/>
     </channel-type>
-
+    <channel-type id="setpointstatus">
+        <item-type>String</item-type>
+        <label>Setpoint Status</label>
+        <description>Thermostat setpoint status</description>
+        <category>heating</category>
+        <state readOnly="true"/>
```

Comment:
I can't reply to yours so I'll leave it here: you can probably still have a separate channel for setting and reading in the binding itself. It's just a channel, quite literally